src/main/java/bio/terra/service/filedata/azure/AzureSynapsePdao.java

```diff
@@ -196,11 +203,16 @@ public int createParquetFiles(
     sqlCreateTableTemplate.add("isCSV", isCSV);
     if (isCSV) {
       sqlCreateTableTemplate.add("parserVersion", PARSER_VERSION);
-      if (csvSkipLeadingRows.isEmpty()) {
+      if (csvSkipLeadingRows == null) {
         throw new InvalidJobParameterException(
             "For CSV ingests, 'csvSkipLeadingRows' must be defined.");
       }
-      sqlCreateTableTemplate.add("firstRow", csvSkipLeadingRows.get());
+      sqlCreateTableTemplate.add("firstRow", csvSkipLeadingRows);
```

Comment:
If you don't mind throwing `NullPointerException`, you can replace with
```
      sqlCreateTableTemplate.add("firstRow", Objects.requireNonNull(csvSkipLeadingRows, "For CSV ingests, 'csvSkipLeadingRows' must be defined."));
```

If you don't want to throw an NPE, you can replace with
```
      sqlCreateTableTemplate.add("firstRow", Objects.requireNonNullElseGet(csvSkipLeadingRows, () -> { throw new InvalidJobParameterException("For CSV ingests, 'csvSkipLeadingRows' must be defined." })));
```